common/config/babel/.babelrc.js

```diff
@@ -26,7 +26,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // PSTN calls 
         'PSTN-calls',
         // props to allow Contoso to overwrite timestamp format for chat messages, one in locale and one in message thread component
-        'date-time-customization'
+        'date-time-customization',
+        // 1 to N Calling feature.
+        '1-n-calling'
```

Comment:
Heh. Even though I thoroghly know how the conditional compilation is implemented and know that it is just string matching... something primal in me recoils at a flag starting with a numeral. :)

Make it `one-to-n-calling` and I'll get you an extra coffee.

(This comment is about 98% a joke)

Reply:
As you desire :P